recipes/clickclick/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.2.1" %}
+
+package:
+  name: clickclick
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/zalando/python-clickclick/archive/{{ version }}.tar.gz
+  sha256: f29b1af9cf5a388e56e2c2aab6bafb4e1bec6f0424a53fa77ba83f4646f09910
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
```

Comment:
it looks like all you need in the `build` section are `python` and `setuptools`, `six` and `flake8` if you use:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26